mozci/console/commands/push.py

```diff
@@ -225,12 +226,54 @@ def handle(self) -> None:
                 "<comment>Provided --output pointed to a inexistent directory that is now created.</comment>"
             )
 
+        try:
+            backfill_limit = int(self.option("backfill-limit"))
+        except ValueError:
+            raise Exception("Provided --backfill-limit should be an int.")
+
         for push in pushes:
             try:
-                classification, regressions = push.classify(**classify_parameters)
+                classification, regressions, to_retrigger_or_backfill = push.classify(
+                    **classify_parameters
+                )
                 if retrigger_unknown:
                     for _, tasks in regressions.unknown.items():
                         retrigger(tasks=tasks, repeat_retrigger=1)
+
+                def retrigger_failures(groups, count):
+                    groups_with_failures = {
+                        name: failing_tasks
+                        for name, failing_tasks in groups.items()
+                        if failing_tasks
+                    }
+                    if not groups_with_failures:
+                        return
+
```

Comment:
Could you assert here that all failing tasks did fail?

Reply:
Not really necessary at the moment, but better to be defensive to potential future changes given that retriggers and backfills can be very expensive.